scene/gui/scroll_container.cpp

```diff
@@ -512,6 +517,50 @@ void ScrollContainer::set_follow_focus(bool p_follow) {
 	follow_focus = p_follow;
 }
 
+float ScrollContainer::get_scroll_sensitivity() const {
+	return sensitivity;
+}
+
+void ScrollContainer::set_scroll_sensitivity(float _sensitivity) {
+	if (_sensitivity < 0.0) {
+		WARN_PRINT("scroll_wheel_sensitivity cannot be negative");
+		_sensitivity = 0.0;
+	}
```

Comment:
This can be an error condition instead:
```suggestion
	ERR_FAIL_COND(p_sensitivity < 0.0);
```